frontend/src/breeding_form.tsx

```diff
@@ -419,10 +441,29 @@ export function BreedingForm({
               )}
               // jscpd:ignore-end
 
-              onChange={(event: any, newValue: OptionType | null) => {
-                newValue && setFormField("father", newValue.value);
+              onChange={(event: any, newValue: LimitedIndividual | null) => {
+                newValue && setFormField("father", newValue.number);
               }}
             />
+            {is_admin ||
+              (is_owner && (
+                <KeyboardDatePicker
+                  autoOk
+                  variant="inline"
+                  inputVariant={inputVariant}
+                  disableFuture={false}
```

Comment:
Why would we allow future dates? That doesn't make sense in this context, does it?